cmd/sonobuoy/app/args.go

```diff
@@ -50,6 +50,26 @@ func AddNamespaceFlag(str *string, flags *pflag.FlagSet) {
 	)
 }
 
+// AddDNSNamespaceFlag initialises the dns-namespace flag.
+// The value of this flag is used during preflight checks to determine which namespace to use
+// when looking for the DNS pods.
+func AddDNSNamespaceFlag(str *string, flags *pflag.FlagSet) {
+	flags.StringVar(
+		str, "dns-namespace", config.DefaultDNSNamespace,
+		"The namespace to check for DNS pods during preflight checks.",
+	)
+}
+
+// AddDNSPodLabelsFlag initialises the dns-pod-labels flag.
+// The value of this flag is used during preflight checks to determine which labels to use
+// when looking for the DNS pods.
+func AddDNSPodLabelsFlag(str *[]string, flags *pflag.FlagSet) {
+	flags.StringArrayVar(
+		str, "dns-pod-labels", config.DefaultDNSPodLabels,
+		"The label selectors to use for locating DNS pods during preflight checks.",
```

Comment:
Can we clarify in the help how to specify multiple ones? Its a comma-delimited list?

Reply:
Ahh, I'm so glad you asked me about this. It encouraged me to do a bit more testing locally and realised I was using the wrong pflag function and that it didn't do what I thought it did!